src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -304,3 +305,40 @@ def session_duration_filters(org_id):
             ),
         )
     ]
+
+
+def histogram(
+    aggregate_filter,
+    histogram_buckets: int = 100,
+    histogram_from: Optional[float] = None,
+    histogram_to: Optional[float] = None,
+    alias=None,
+):
+    zoom_conditions = zoom_histogram(
+        histogram_buckets=histogram_buckets,
+        histogram_from=histogram_from,
+        histogram_to=histogram_to,
+    )
+    if zoom_conditions is not None:
+        conditions = Function("and", [zoom_conditions, aggregate_filter])
+    else:
+        conditions = aggregate_filter
+
+    return Function(
+        "histogramIf(250)",
```

Comment:
is it something we would like to hardcode or extract as constant?

Reply:
I feel like this is okay because the function generating is called histogram if you mean the on `histogramIf`? if you are talking about the 250, we actually want that hardcoded

Reply:
yes `250`. Okey but i saw it here:
```
src/sentry/snuba/metrics/fields/histogram.py
9:MAX_HISTOGRAM_BUCKET = 250
```

Reply:
agreed, reused the constant